Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+LABEL maintainer "Wes Lambert, wlambertts@gmail.com"
```

Comment:
We have used "author" rather than "maintainer" in our other files. please could you change this to "author" for consistency?

Reply:
Absolutely!

Reply:
Updated in PR.